modules/postgresql/src/main/java/org/testcontainers/containers/PostgreSQLContainer.java

```diff
@@ -73,4 +80,23 @@ public SELF withPassword(final String password) {
         this.password = password;
         return self();
     }
+
+    @Override
+    protected void waitUntilContainerStarted() {
+        String regEx = ".*database system is ready to accept connections.*\\s";
+        logger().info("Waiting for database to log '{}'", regEx);
+
+        WaitingConsumer waitingConsumer = new WaitingConsumer();
+        this.followOutput(waitingConsumer);
+
+
+        Predicate<OutputFrame> waitPredicate = outputFrame ->
+                outputFrame.getUtf8String().matches(regEx);
```

Comment:
I've added the change in this PR.
Can someone with write access please retry the TravisCI build? The same build was green in my branch, so I think this might have been some problem with Travis or Docker in the build.